.github/workflows/backport.yml

```diff
@@ -43,6 +43,15 @@ jobs:
               name: 'backport: auto'
             })
 
+  list-target-branches:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.get-branch-list.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: get-branch-list
+        run: echo "::set-output name=matrix::$(yq '.target_branch_list' ./detection_rules/etc/target-branches.yml)"
+
```

Comment:
Looking for a way to remove this duplicated job and reference it from `branch-status-checks.yml`

Reply:
Was able to use https://docs.github.com/en/actions/using-workflows/reusing-workflows